### DISCUSSION TITLE
Bale collector Guide

### DISCUSSION THREAD

**Discussion by landtools** [Courseplay/courseplay#7511] [Q&A]
hi, do we have a guide on how to make the bale collector function work ??

**Comment by Tensuko** [comment]
#6859
For what do we do stickies ?

**Reply by landtools** [reply]
read ..... ok, after creating the route, with start and finish in the field, the trailer does not unload ....what am I doing wrong?
![20210926_165502](https://user-images.githubusercontent.com/71590173/134813096-3d66fc84-8b7e-4ddd-9dc4-d2d47d1267b3.jpg)

**Comment by Tensuko** [comment]
For trigger, do not use the unload WP.
Not sure if that Trailer can unload correct.

**Reply by landtools** [reply]
ok, I removed the WP but it goes on anyway .. I try to change Trailer ..
